### PR TITLE
Section Selector Styling

### PR DESCRIPTION
**Pull request by sigmachirality**
I think in general, the selector needs to be made wider. Right now it's hard to style the selector to follow Jess's design with the space given. Currently the selector looks something like this:

Screenshot from 2019-10-17 23-55-06

If we want to integrate in PCA we'll need more space to add another column in.

### CODE REVIEW

**Comment by davish** [comment]
[Course detail pop-down](https://trello.com/c/cQ7JbJiI/64-course-detail-pop-down)

**Comment by davish** [comment]
[Styling on Section Selector](https://trello.com/c/9Y1ZbC4l/67-styling-on-section-selector)

File: frontend/plan/src/components/selector/CourseDetails.js [hunk 1]
```diff
@@ -1,21 +1,37 @@
...
+                    value={course.difficulty}
+                />
+            </li>
             <li>
                 <span className="icon is-small">
                     <i className="far fa-check-circle" />
                 </span>
                 &nbsp; Fulfills: &nbsp;
-                { <TagList elements={requirements.map(req => `${req.school}: ${req.name}`)} limit={1} /> }
+                { <TagList elements={requirements.map(({ school, name }) => `${school.charAt(0)}: ${name}`)} limit={1} /> }
```
[the remark this answers is not in the document]
Comment (sigmachirality) [reply frontend/plan/src/components/selector/CourseDetails.js]: probably can add that to the utils file

**Comment by sigmachirality** [comment]
fixed in davish merge dan4 (pls work)

**Pull request closed without merging**